html-and-the-web/html-and-the-web_div-salad/index.html

```diff
@@ -15,49 +15,50 @@
 	which one was it again?-->
   <body>
 
-    <div>
+    <header>
       <div>
-        <div>Jane Doe</div>
+        <h1>Jane Doe</h1>
         <!--Which elements could be used to link to parts of the webpage,
         which appear below?-->
         <div id="nav">
-          <div>About Me</div>
-          <div>Contact</div>
-          <div>Skillset</div>
+          <a href="about-me">About Me</a>
```

Comment:
add the # in front of the section names: i.e. 
```<a href="#about-me">About Me</a>```

Reply:
Thank you!